spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/StoredProcJavaConfigTests.java

```diff
@@ -77,7 +77,8 @@
 	public void test() {
 		Message<?> received = fooChannel.receive(10000);
 		assertThat(received).isNotNull();
-		Collection<?> primes = (Collection<?>) received.getPayload();
+		@SuppressWarnings("unchecked")
+		Collection<Integer> primes = (Collection<Integer>) received.getPayload();
 		assertThat(primes).containsExactly(2, 3, 5, 7);
```

Comment:
We don't fail here even right now even my IDEA doesn't like wildcards as well:

![image](https://user-images.githubusercontent.com/1071590/53659711-34953f00-3c2a-11e9-9b88-02f5a4a52cc9.png)

I wonder what is wrong then...

Reply:
A couple of days ago, I think I tweaked a setting in STS to be more strict, because it just started complaining about this.